policyengine_us/parameters/gov/states/ri/tax/income/exemption/reduction/percentage.yaml

```diff
@@ -0,0 +1,41 @@
+description: Rhode Island phases this percentage of the total exemption amount based on modified agi.
+metadata:
+  type: single_amount
+  threshold_unit: /1
+  amount_unit: /1
+  label: Rhode Island exemption phase out rate 
+  reference:
+  # The tax form points to this worksheet: Exemption Worksheet for RI-1040NR, Page 1, line 6)
+  - title: Rhode Island Tax Rate Schedule and Worksheet 2022 (Exemption Worksheet line 7)
+    href: https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/2022%20Tax%20Rate%20and%20Worksheets.pdf
+  # The tax form points to this worksheet: Exemption Worksheet for RI-1040 or RI-1040NR, Page 1, line 6)
+  - title: Rhode Island Tax Rate Schedule and Worksheet 2021 (Exemption Worksheet line 7)
+    href: https://tax.ri.gov/sites/g/files/xkgbur541/files/2021-11/2021-tax-rate-and-worksheets.pdf
+  - title: Rhode Island Legal Code R.I. Gen. Laws § 44-30-2.6 (E), (4), (b)
+    href: http://webserver.rilin.state.ri.us/Statutes/TITLE44/44-30/44-30-2.6.HTM
```

Comment:
This is what this says:
>In the case of any taxpayer whose adjusted gross income for the taxable year exceeds the threshold amount, the exemption amount shall be reduced by two (2) percentage points for each $2,500 (or fraction thereof) by which the taxpayer's adjusted gross income for the taxable year exceeds the threshold amount. In the case of a married individual filing a separate return, the preceding sentence shall be applied by substituting ''$1,250'' for ''$2,500.'' In no event shall the applicable percentage exceed one hundred percent (100%).

Nowhere here are the numbers in this file. Never tie a reference that do not reflect the values in the parameters.

You need to find what law allows the tax form to increment by 20 percentage points. We should be able to reflect this as a single number rather than a scale, as the law does.